osu.Game.Rulesets.Osu/Objects/OsuHitObject.cs

```diff
@@ -16,14 +16,17 @@ public abstract class OsuHitObject : HitObject, IHasCombo, IHasPosition
         public const double OBJECT_RADIUS = 64;
 
         private const double hittable_range = 300;
-        private const double hit_window_50 = 150;
-        private const double hit_window_100 = 80;
-        private const double hit_window_300 = 30;
+        private static double hitWindow50 = 150;
+        private static double hitWindow100 = 80;
+        private static double hitWindow300 = 30;

```

Comment:
Why are these statics needed? `OsuHitObject.HitWindowFor()` has them already.

Reply:
this is a mistake, i will revert it